README.md

```diff
@@ -81,9 +81,10 @@ Either the tab or cli argument is required to output lyrics.
 
 Arguments:
 ```
-  -h, --help  show this help message and exit
-  -t, --tab   Display lyrics in a browser tab.
-  -c, --cli   Display lyrics in the command-line.
+  -h, --help    show this help message and exit
+  -t, --tab     Display lyrics in a browser tab.
+  -c, --cli     Display lyrics in the command-line.
+  -cr, --chrome fetch Lyrics from browser (to be passed along those above when required) 
```

Comment:
Since --chrome is not a standalone argument and is to be used in addition to -t or -c, it should just be --chrome and no -cr.